belt-node-notifications-hub.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+# modifications to the base-ci-resource GitLab CI file to insert Notifications Hub system tests between deployment to INT and PRD
```

Comment:
This is an exact copy of `belt-node8-notifications-hub.gitlab-ci.yml`, which I intend to delete once all the NH repos are moved over to this one.